package.json

```diff
@@ -16,6 +16,7 @@
     "gen-data-json": "node hack/build-template-data.js",
     "prebuild": "rm -fr dist/* && npm run gen-data-json",
     "build": "tsc && node build.mjs",
+    "build:image": "npm run test:journey:build && docker buildx build --tag pepr:dev .",
```

Comment:
I kinda don't like having a generic `build:image` command rely on the more-specialized `test:journey:build` script -- it seems a little confused to have a central-to-the-program script rely on a test script to me; feels snarly. :thinking: 

This change also results in a duplication of the docker CLI code for building "the dev image" (here & down in "test:journey:k3d") -- a task there is really only "one right way" to do but 2 separate code blocks performing independently.

The GitHubs PR UI is being weird and not letting me make a suggestion directly below, but what would you think of just updating the "test:journey:k3d" step something like:

```
    "test:journey:image-build": "docker buildx build --tag pepr:dev .",
    "test:journey:image-import": "k3d image import pepr:dev -c pepr-dev",
    "test:journey:image": "npm run test:journey:image-build && npm run test:journey:image-import",
```

And just using the split-out "test:journey:image-build" command instead of `build:image`?  It cuts the duplication & still gets us an independently-callable image build script?

Reply:
Okay, looking to follow but may be lost.

so
Step 1 -  you want to move `npm run test:journey:build` to ci?
Step 2 - create a new npm script for `docker buildx build --tag pepr:dev .` with a more detailed name?

Reply:
chatted sync and we decided to remove `npm pack` to make more efficient